packages/desktop-client/src/components/reports/graphs/CashFlowGraph.tsx

```diff
@@ -26,7 +26,7 @@ function CashFlowGraph({ graphData, isConcise }: CashFlowGraphProps) {
         graphData && (
           <VictoryChart
             scale={{ x: 'time', y: 'linear' }}
-            theme={theme}
+            theme={chartTheme}
```

Comment:
Nothing in that file has changed. I just changed the reference name because the style "theme" was the same.

Reply:
This would be my only guess.

What I would recommend doing is: run the VRT tests locally. Revert a few changes.. run them again. Rinse and repeat until you have narrowed down the change that causes the breakage.

Reply:
So I've compared the master on my dev server to this PR and I don't see this regression showing up. Is it possible the VRT is wrong?

Reply:
If the VRT was wrong you'd see it failing on other PRs too with the same problem. But it's not the case (example: https://github.com/actualbudget/actual/pull/1692). So must be something in your PR that's causing it to fail.

Reply:
Okay. 

Last question: It could take many hours of searching for this via manual VRT checks. It is a really tiny unnoticeable regression that doesn't really change the functionality or the look of the app. It's even something that is used in other parts of the app (a same color shadow or border on an element). With the time gained I would be able to finish the last dark theme PR and we could be done with them and move on to more important fixes/changes.

Thoughts on keeping it as is?

Reply:
I'm more worried about the unknown-unknown here. If there is a regression here and we don't know why: what other things are we missing as well? 

It really shouldn't be hours of efforts as long as you run them locally. On avg. the whole test suite takes me 4 minutes. If it's isolated to just a single test, then it's < 1 minute.

```
E2E_TEST_URL=... yarn vrt reports
```

Reply:
Okay, found something to do with the export and reference of the chart-theme. I've updated the code to work more efficiently. As a result it's also remedied the VRT error.